cpu/cc26x0/include/cc26x0.h

```diff
@@ -85,6 +85,7 @@ typedef enum IRQn
     IRQN_COUNT            = (TRNG_IRQN + 1) /**< Number of peripheral IDs */
 } IRQn_Type;
 
+
```

Comment:
regarding this commit:
i will merge it but i can't say whether it'll make its way upstream.